src/hotspot/share/gc/shenandoah/heuristics/shenandoahStaticHeuristics.hpp

```diff
@@ -27,6 +27,9 @@
 
 #include "gc/shenandoah/heuristics/shenandoahHeuristics.hpp"
 
+// TODO: Make this a subclass of ShenandoahHeuristics and create a "tailored" copy of this class to be subclass of
+// ShenandoahOldHeuristics.  Fix the constructor implementation to invoke ShenandoahHeuristics() superclass constructor.
+// Change ShenandoahMode::initialize_old_heuristics() to return the subclass of ShenandoahOldHeuristics.
```

Comment:
Is this comment still valid? You seem to propose a ShenandoahStaticOldHeuristics too.

Reply:
Oops.  Forgot to remove that.  Removed in next commit.